message_definitions/v1.0/common.xml

```diff
@@ -2884,6 +2884,79 @@
         <description>Use precision landing, searching for beacon if not found when land command accepted (land normally if beacon cannot be found).</description>
       </entry>
     </enum>
+    <enum name="OPENDRONEID_ID_TYPE">
+      <entry value="0" name="ODID_IDTYPE_NONE">
+        <description>No type defined.</description>
+      </entry>
+      <entry value="1" name="ODID_IDTYPE_SERIAL_NUMBER">
+        <description>Serial Number (ANSI/CTA-2063).</description>
+      </entry>
+      <entry value="2" name="ODID_IDTYPE_CAA_ASSIGNED_ID">
+        <description>CAA (Civil Aviation Authority) Assigned ID.</description>
+      </entry>
+      <entry value="3" name="ODID_IDTYPE_UTM_ASSIGNED_ID">
+        <description>UTM (Unmanned Traffic Management) Assigned ID (UUID RFC4122).</description>
+      </entry>
+    </enum>
+    <enum name="OPENDRONEID_UAS_TYPE">
+      <entry value="0" name="ODID_UASTYPE_NONE">
+        <description>No UAS (Unmanned Air/Aircraft System) type defined.</description>
+      </entry>
+      <entry value="1" name="ODID_UASTYPE_FIXED_WING_POWERED">
+        <description>Fixed Wing Powered.</description>
+      </entry>
+      <entry value="2" name="ODID_UASTYPE_ROTORCRAFT_MULTIROTOR">
+        <description>Rotorcraft/Multirotor.</description>
```

Comment:
This is *so* going to annoy the Helicopter people :-)

It's possibly actually important from a safety viewpoint; helicopters can move and navigate significantly differently compared to a multirotor.

I expect a response to this comment of "that's what's in the spec" :-)

Reply:
Indeed :-) That is what the Open Drone ID spec currently specify. We would need to take this up with Gabriel Cox, if a change is needed.